plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.tsx

```diff
@@ -143,11 +143,25 @@ export const StepPrepareCreatePullRequest = (
       .sort();
   });
 
+  const isValidObjectName = (value: unknown): boolean => {
+    return (
+      typeof value === 'string' &&
+      value.length >= 1 &&
+      value.length <= 63 &&
+      /^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]$/.test(value)
+    );
+  };
```

Comment:
This would need to rely on the remote catalog validation in some way. The validation logic that the catalog backend uses can be reconfigured, so we can't hardcode it here.